f8a_worker/dispatcher/flows/victimsFlow.yml

```diff
@@ -0,0 +1,11 @@
+---
+  flow-definitions:
+    - name: 'victimsFlow'
+      queue: '{DEPLOYMENT_PREFIX}_{WORKER_ADMINISTRATION_REGION}_victimsFlow_v0'
+      sampling:
+        name: 'constant'
+        args:
+          retry: 20
+      edges:
+        - from:
```

Comment:
So this flow will be started just from jobs service right?

Reply:
My idea is to trigger it (via jobs) when we push something to master branch.